docs/source/Advanced-Usage.md

```diff
@@ -189,9 +204,12 @@ Refer to the following blog post for an example on how to deploy IntelOwl on Goo
 [Deploying Intel-Owl on GKE](https://mostwanted002.cf/post/intel-owl-gke/) by [Mayank Malik](https://twitter.com/_mostwanted002_).
 
 ## Multi Queue
-IntelOwl provides a [docker-compose-multi-queue](https://github.com/intelowlproject/IntelOwl/blob/master/docker-compose-multi-queue.yml), allowing IntelOwl users to better scale with the performance of their own architecture.
+IntelOwl provides an additional `docker-compose` file,  [multi-queue.override.yaml](https://github.com/intelowlproject/IntelOwl/blob/master/docker-compose-multi-queue.yml) file, allowing IntelOwl users to better scale with the performance of their own architecture.
```

Comment:
need to change the name and and link to point to `docker/multi-queue.override.yml`

Reply:
same for line 212